modules/cli-authentication.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * cli/using-roxctl-cli.adoc
+:_module-type: PROCEDURE
+[id="cli-authentication_{context}"]
+= Authenticating using the roxctl CLI
+
+For authentication, you can use an authentication token or your administrator password.
+Red Hat recommends using an authentication token in a production environment because each token is assigned specific access control permissions.
+//TODO: Add links to role based access control
+
+Use the following steps to generate an authentication token.
+
+.Procedure
+
+. Navigate to the RHACS portal.
+. Go to *Platform Configuration* -> *Integrations*.
+. Scroll down to the *Authentication Tokens* category, and click *API Token*.
+. Click *Generate Token*.
+. Enter a name for the token and select a role that provides the required level of access (for example, *Continuous Integration* or *Sensor Creator*).
+. Click *Generate*.
++
+[IMPORTANT]
+====
+Copy the generated token and securely store it.
+You will not be able to view it again.
+====
+
+[NOTE]
+====
+After you have generated the authentication token, export it as `ROX_API_TOKEN` variable:
+[source,terminal]
+----
+$ export ROX_API_TOKEN=<api-token>
```

Comment:
We use underscores between mutli-word replaceable values, if you don't mind updating here and in the below command.

There are a few other modules that also need updating, if you don't mind doing a check through.